WordPress/WordPressTest/BlogServiceRemoteRESTTests.m

```diff
@@ -1,8 +1,13 @@
 #import <OCMock/OCMock.h>
+#import <OHHTTPStubs/OHHTTPStubs.h>
 #import <XCTest/XCTest.h>
 #import "Blog.h"
 #import "BlogServiceRemoteREST.h"
 #import "WordPressComApi.h"
+#import "WordPress-Swift.h"
+#import "WordPressTests-Swift.h"
+
+static NSTimeInterval const TestExpectationTimeout = 2;
```

Comment:
This is only used for a local async OP, but you're right, just for safety, let's bump this one up. Thanks!